### PR TITLE
Fix Identical Icons MDFloatingActionButtonSpeedDial

### PR DESCRIPTION
**Pull request by VICTORVICKIE**
### Description of Changes
Checked with KitchenSink and everything works as expected and it allows users/dev to use identical icons in speed dials

data provided should be as follows,

### Examples

```
data = {'Python':'language-python' ,
            'PHP':'language-php' ,
            'C++':'language-cpp' ,
           }
```
Screenshot_3
```
data = {'Python':'plus' ,
            'PHP':'plus' ,
            'C++':'plus' ,
           }
```
Screenshot_2

### CODE REVIEW

**Comment by HeaTTheatR** [comment]
@VICTORVICKIE 
Please write more informative commits. What exactly did you do and why.

Снимок экрана 2021-03-21 в 17 12 53

**Comment by VICTORVICKIE** [comment]
> @VICTORVICKIE
> Please write more informative commits. What exactly did you do and why.
> 
> Снимок экрана 2021-03-21 в 17 12 53

Sorry, now I've added informative commit message

File: demos/kitchen_sink/libs/baseclass/stack_buttons.py [hunk 1]
```diff
@@ -15,6 +15,7 @@ def callback(instance_button):
 
             button_speed_dial = MDFloatingActionButtonSpeedDial()
             button_speed_dial.root_button_anim = True
+            # {"Text to be displayed" : "Icon name"}
```
Comment (HeaTTheatR) [inline demos/kitchen_sink/libs/baseclass/stack_buttons.py]: Please remove these comments - they are not needed in this context

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

File: kivymd/uix/button.py [hunk 2]
```diff
@@ -1872,6 +1872,9 @@ def on_data(self, instance, value):
         self._label_pos_y_set = False
 
         # Bottom buttons.
+        # Added New format for data of speed dial for identical icons
+        # {"Text to be displayed":"icon name"}
+
```
Comment (HeaTTheatR) [inline kivymd/uix/button.py]: Please remove these comments - they are not needed in this context

**Review by HeaTTheatR — CHANGES_REQUESTED** [review]

**Comment by HeaTTheatR** [comment]
@VICTORVICKIE 
Please, refactor your code - https://github.com/kivymd/KivyMD#setup-environment

Снимок экрана 2021-03-21 в 18 16 46

**Pull request merged**